demos/dropbase.ftd

```diff
@@ -38,100 +38,208 @@ custom tools with SQL and Python in minutes.
+
+-- end: ds.inline-row
 
 -- end: ds.section-row
 
+-- end: ftd.desktop
+
+-- ftd.mobile:
+
+-- ds.section-column:
+;;background.solid: $ds.colors.background.step-1
+spacing: $ds.spaces.vertical-gap.space-between
+;;margin: $ds.spaces.vertical-gap.extra-large
+inset: $ds.spaces.inset-wide.large
+align-content: top-left
+
+-- ds.row:
+spacing: $ds.spaces.vertical-gap.space-between
+align-content: center
+
+-- ftd.image:
+src: $assets.files.assets.logo.svg
+width.fixed.px: 150
+
+-- ftd.image:
+src: $assets.files.assets.menu.svg
+width.fixed.px: 24
+$on-click$: $ftd.toggle($a = $header.open)
+
+-- end: ds.row
+
+-- ds.column:
+if: {header.open}
+anchor: parent
+top: 72
+bottom: 0
+left: 0
+right: 0
+height: hug-content
+background.solid: $ds.colors.background.step-1
+z-index: 1
+$on-click$: $ftd.toggle($a = $header.open)
+align-content: top-left
+inset: $ds.spaces.inset-square.medium
+
+-- ds.column:
+inset: $ds.spaces.inset-wide.medium
+align-content: top-left
+
+-- ds.header-link: Docs
+link: /
+
+-- ds.header-link: Blog
+link: /
+
+-- ds.header-link: Pricing
+link: /
+
+-- ds.header-link: Content
+link: /
+
+-- end: ds.column
+
+-- end: ds.column
+
+-- end: ds.section-column
+
+-- end: ftd.mobile
+
+-- end: ds.column
+
 -- end: header
 
 
 
 
 
 
+
+
+
+
 -- component hero:
 
+-- ds.column:
+
+-- ftd.desktop:
+
 -- ds.section-row: hero section
-;; this doesn't work
-;; margin: $ds.spaces.vertical-gap.extra-extra-large
-;; following both work
-;; margin.px: 128
-;; TODO
 margin: $ds.spaces.vertical-gap.extra-extra-large
 
+-- ds.column: left column
+
+-- ds.heading-tiny: DROPBASE
+
+-- ds.heading-large: Unlock insights from Conversations using GenAI
 
-	-- ds.column: left column
-	
-		-- ds.heading-tiny: DROPBASE
-		
-		-- ds.heading-large: Unlock insights from Conversations using GenAI
-		
-		-- ds.copy-large:
-		
-		Leverage AI driven insights & actionables from your customer conversations
-		across all channels - Calls, Chats & Meetings to boost your business efficiency
-		
-		-- ds.button-group:
-		
-			-- ds.primary-button: Book Demo
-			
-			-- ds.phosphor-icon-button: Documentation
-			icon: caret-right
-			icon-position: at-end
-			
-		-- end: ds.button-group
+-- ds.copy-large:
 
-	-- end: ds.column
+Leverage AI driven insights & actionables from your customer conversations
+across all channels - Calls, Chats & Meetings to boost your business efficiency
 
+-- ds.button-group:
 
+-- ds.primary-button: Book Demo
 
-	-- ds.column: right column
-	
-		-- ftd.image: https://assets-global.website-files.com/5f2c87246b17fcf662282594/651dce0aa7133cdfef4c9431_Dropbase%20Python.webp
-		width: fill-container
-		
-	-- end: ds.column
+-- ds.phosphor-icon-button: Documentation
+icon: caret-right
+icon-position: at-end
 
+-- end: ds.button-group
 
+-- end: ds.column
+
+-- ds.column: right column
+
+-- ftd.image: https://assets-global.website-files.com/5f2c87246b17fcf662282594/651dce0aa7133cdfef4c9431_Dropbase%20Python.webp
```

Comment:
Do checking all these files to our repo.

Reply:
Actually, do not copy exact image, put some other image here  that we have copy right over.